src/components/Placeholders/PaperDefinitionsPropTypes.js

```diff
@@ -5,8 +5,8 @@ const paperDefinitionsPageAttrProptypes = PropTypes.shape({
   type: PropTypes.string.isRequired
```

Comment:
:question: Should `paperDefinitionsPageAttrProptypes` be renamed as `paperDefinitionsStepAttrProptypes`?